docs/Images.md

```diff
@@ -4,7 +4,7 @@ title: Images
 layout: docs
 category: Guides
 permalink: docs/images.html
-next: gesture-responder-system
+next: colors
```

Comment:
I also changed this 'next' link in #8299 (to handling-touches), and we should coordinate to make sure these two PRs do not conflict.

I propose that Style.md be updated to point to `colors` in its next link, and Colors.md be updated to point to 'images', resulting in:

* Style
* Colors
* Images
* Handling Touches

What do you think?